tests/h/notification/reply_test.py

```diff
@@ -148,11 +145,11 @@ def fetch_annotation(self, patch, annotations):
         return fetch_annotation
 
     @pytest.fixture(autouse=True)
-    def subscription(self, db_session):
-        sub = Subscriptions(type="reply", active=True, uri="acct:giraffe@safari.net")
-        db_session.add(sub)
-        db_session.flush()
-        return sub
+    def subscription_service(self, subscription_service, factories):
+        subscription_service.get_subscription.return_value = factories.Subscriptions(
+            active=True, type=Subscriptions.Type.REPLY.value
+        )
```

Comment:
This is an annoyingly common pattern, to the point where you _might_ be tempted to put this sort of thing into the underlying fixture, but I don't think it's a good idea:

 * The principal of least surprise suggests all these fixtures are nicer to work with when they don't all have custom behavior
 * It's easy to add the behavior you want, but it's hard to get back to a vanilla mock if you don't want it
 * There are quite a few different methods we'd have to implement